src/components/Popups/TransactionPopup.tsx

```diff
@@ -15,21 +19,34 @@ const RowNoFlex = styled(AutoRow)`
   margin-right: 16px;
 `
 
-export function TransactionPopup({ hash, success, summary }: PopupContent) {
+export function TransactionPopup({ hash, success, summary, swapProtocol }: PopupContent) {
   const { chainId } = useActiveWeb3React()
 
+  const { t } = useTranslation()
   const theme = useTheme()
 
+  const isGnosisProtocolHash = swapProtocol === SwapProtocol.COW
+
+  const explorer = isGnosisProtocolHash ? t('gnosisProtocolExplorer') : t('blockExplorer')
+
+  const href = isGnosisProtocolHash
```

Comment:
I think we should be consistent on the naming, we used link before and I think `link` sounds better